packages/seacas/libraries/nemesis/CMakeLists.txt

```diff
@@ -28,17 +28,13 @@ TRIBITS_ADD_LIBRARY(
   SOURCES ${SOURCES}
   )
 
-IF (${PACKAGE_NAME}_ENABLE_TESTS)
-  TRIBITS_ADD_TEST(
+IF (${CMAKE_PROJECT_NAME}_ENABLE_TESTS)
```

Comment:
Why was this changed?  The correct logic **is** `if (${PACKAGE_NAME}_ENABLE_TESTS)`.

This change violates the documented behavior of TriBITS projects (see [<Project>_ENABLE_TESTS only enables explicitly enabled package tests](https://tribitspub.github.io/TriBITS/users_guide/index.html#project-enable-tests-only-enables-explicitly-enabled-package-tests)) .  For example, if someone configures Trilinos with:

```
-D Trilinos_ENABLE_Panzer=ON \
-D Trilinos_ENABLE_TESTS=ON \
```

they are supposed to only get Panzer tests and examples turned on.  But with checking only `${CMAKE_PROJECT_NAME}_ENABLE_TESTS`, these tests would be enabled by surprise to everyone.

However, note that this if statement is meaningless because the the logic to enable/disable tests is baked into the functions `tribits_add_test()` and `tribits_add_executable_and_test()`.  See:

* https://tribitspub.github.io/TriBITS/users_guide/index.html#tribits-add-test

which says:

> The tests are only added if tests are enabled for the package (i.e. [`${PACKAGE_NAME}_ENABLE_TESTS`](https://tribitspub.github.io/TriBITS/users_guide/index.html#package-name-enable-tests))

(I need to change the last part of that sentence because it is no longer true that setting `${PARENT_PACKAGE_NAME}_ENABLE_TESTS=TRUE` will automatically enable the tests in all enabled subpackages.)

Reply:
I have removed the `if` block to let `TRIBITS_ADD_EXECUTABLE_AND_TEST` take care of enable/disable.